packages/theming/theme-types/src/Theme.types.ts

```diff
@@ -6,12 +6,26 @@ import { PartialShadowDefinition, ThemeShadowDefinition } from './Shadow.types';
 
 type TwoLevelPartial<T> = { [K in keyof T]?: Partial<T[K]> };
 
+type SpacingValue = number | string;
+
 export interface Spacing {
   s2: string;
   s1: string;
-  m: string;
+  // m is now defined as a v2 type
   l1: string;
   l2: string;
+  // v2 types
+  none?: SpacingValue;
```

Comment:
Thanks for this, I should probably get the win32 spacing to come through here too instead of pulling directly from global tokens, which is what we do today

Reply:
It does mean that clients are able to change spacing which we generally don't allow for global tokens though.. hm.

Reply:
Do you know if design has settled on spacing having alias tokens?

Reply:
As far as I can tell, spacing doesn't have alias tokens.

Reply:
I would be interested in this conversation, I am putting up a similar change for Android and if spacing needs to be put elsewhere outside Theme to be non modifiable by clients - what would be a good way to handle this..?